b2sdk/v1/sync/report.py

```diff
@@ -0,0 +1,33 @@
+######################################################################
+#
+# File: b2sdk/v1/sync/report.py
+#
+# Copyright 2021 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+from b2sdk import _v2 as v2
+
+
+# override to retain legacy methods
+class SyncReport(v2.SyncReport):
+    @property
+    def local_file_count(self):
+        return self.total_count
+
+    @local_file_count.setter
+    def local_file_count(self, value):
```

Comment:
does this really work? It looks like it would shadow the earlier one

Reply:
what do you mean? The getter and the setter? they don't work if their names are different